code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -64,48 +67,145 @@
+			O.target = user
+			O.t_loc = user.loc
+		else
+			O.target = C
+			O.t_loc = C.loc
+		O.item = user.get_active_hand()
+		O.s_loc = user.loc
+		O.place = "handcuff"
+		C.requests += O
+		spawn()
+			if(istype(src, /obj/item/weapon/handcuffs/cable))
+				feedback_add_details("handcuffs", "C")
+			else
+				feedback_add_details("handcuffs", "H")
+			playsound(get_turf(src), cuffing_sound, 30, 1, -2)
+			O.process()
+			if(mode == SYNDICUFFS_ON_APPLY && !charge_detonated)
+				detonate(1)
+	else
+		var/obj/effect/equip_e/monkey/O = new /obj/effect/equip_e/monkey()
+		O.source = user
+		O.target = C
+		O.item = user.get_active_hand()
+		O.s_loc = user.loc
+		O.t_loc = C.loc
+		O.place = "handcuff"
+		C.requests += O
+		spawn()
+			playsound(get_turf(src), cuffing_sound, 30, 1, -2)
+			O.process()
+			if(mode == SYNDICUFFS_ON_APPLY && !charge_detonated)
+				detonate(1)
+
+/obj/item/weapon/handcuffs/proc/handcuffs_remove(var/mob/living/carbon/C)
+
+	if(mode == SYNDICUFFS_ON_REMOVE && !charge_detonated)
+		detonate(0) //This handles cleaning up the inventory already
+	else
+		C.handcuffed = null
+		C.update_inv_handcuffed()
+
+//Syndicate Cuffs. Disguised as regular cuffs, they are pretty explosive
+/obj/item/weapon/handcuffs/syndicate
+
+	mode = SYNDICUFFS_ON_APPLY
+	var/countdown_time = 30
+
+/obj/item/weapon/handcuffs/syndicate/attack_self(mob/user)
+
+	switch(mode)
+		if(SYNDICUFFS_ON_REMOVE)
+			mode = SYNDICUFFS_ON_APPLY
+			user << "<span class='notice'>You pull the rotating arm back until you hear two clicks. \The [src] will detonate a few seconds after being applied.</span>"
+		if(SYNDICUFFS_ON_APPLY)
+			mode = SYNDICUFFS_ON_REMOVE
+			user << "<span class='notice'>You pull the rotating arm back until you hear one click. \The [src] will detonate when removed.</span>"
+
+//C4 and EMPs don't mix, will always explode at severity 1, and likely to explode at severity 2
+/obj/item/weapon/handcuffs/syndicate/emp_act(severity)
+
+	switch(severity)
+		if(1)
+			if(prob(80))
+				detonate(1)
+			else
+				detonate(0)
+		if(2)
+			if(prob(50))
+				detonate(1)
+
+/obj/item/weapon/handcuffs/syndicate/ex_act(severity)
+
+	switch(severity)
+		if(1)
+			if(!charge_detonated)
+				detonate(0)
+		if(2)
+			if(!charge_detonated)
+				detonate(0)
+		if(3)
+			if(!charge_detonated && prob(50))
+				detonate(1)
+		else
+			return
+
+	qdel(src)
+
+/obj/item/weapon/handcuffs/syndicate/detonate(countdown)
+
+	if(charge_detonated)
+		return
+	charge_detonated = 1 //Do it before countdown to prevent spam fuckery
+	if(countdown)
+		//Cannot use a sleep() instruction here since we do NOT want to delay inventory handling
+		spawn(countdown_time)
+			explosion(get_turf(src), 0, 1, 3, 0)
+			qdel(src)
+	else
+		explosion(get_turf(src), 0, 1, 3, 0)
+		qdel(src)
+
+/obj/item/weapon/handcuffs/Destroy()
+
+	if(iscarbon(loc)) //Inventory shit
+		var/mob/living/carbon/C = loc
+		if(C.handcuffed)
+			C.handcuffed.loc = C.loc //Standby while we delete this shit
```

Comment:
Handled by the proc below, this moves the thing out of the mob's inventory so inventory removal works properly